hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -175,15 +210,15 @@ def test_quant_zeros(self):
                            [0.5, 0.0, 0.0]]).T
         with ignore_warning(message="divide by zero encountered",
                             category=RuntimeWarning):
-            quant = utils_eds.quantification_cliff_lorimer(
-                intens, [1, 1, 3]).T
-        nt.assert_true(np.allclose(
+            quant = utils_eds.quantification(
```

Comment:
This line is wrong -- Pierre was previously using just the actual quantification method specified in utils, you've now written 3 of these quantification_cliff_lorimer etc...

I think you should just switch this back to quantification_cliff_lorimer and then replicate the test for quantification_zeta_factor and quantification_cross_sections